core/src/main/java/org/arquillian/cube/impl/client/container/CubeContainerLifecycleController.java

```diff
@@ -24,7 +24,8 @@
     @Inject
     private Event<CubeControlEvent> controlEvent;
 
-    public void startCubeMappedContainer(@Observes BeforeStart event, CubeRegistry cubeRegistry,
+    // Must be started before ContainerConfigurationController
+    public void startCubeMappedContainer(@Observes(precedence=100) BeforeSetup event, CubeRegistry cubeRegistry,
```

Comment:
Not a fan of this change as it changes the interaction between the Container lifecycle and the 'hidden' Cube'ification of an Adaptor. BeforeSetup is always called in BeforeSuite, while BeforeStart could technically be called at any point controlled by the User in manual mode. With this change, the Container would be started regardless of what the user say..

Cube+Remote adapter actually works very similar to a Managed container at the moment, this change would alter that.

I do understand why you want it in this case, as you can't setup the PortForward until the Pod is running, but... Since we know the Proxy Port we will use, we can technically configure the adapter to use our proxy port before starting the Proxy.